gcloud/storage/test_bucket.py

```diff
@@ -758,6 +758,7 @@ def test_get_logging_w_prefix(self):
         resp_to_reload = before
         connection = _Connection(resp_to_reload)
         bucket = self._makeOne(NAME, connection)
+        bucket._reload_properties()
```

Comment:
Yes. I was going for the fly-by / low-hanging fruit approach of test-fixes.

I can cover it here if you like. Or, instead, I can file a bug to do a standalone test cleanup (essentially just `git grep '.patch'` and `git grep '._reload'`)